### PR TITLE
Update the readme for RC2.

### PR DESCRIPTION
**Pull request by glennc**
Can I get some reviews? @danroth27 @Eilon @shanselman 

What feed should we actually ask people to use here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,41 @@
...
+- Stable builds of ASP.NET Core should be obtained from http://dot.net.
+- If you want to use nightlies then you need to:
+    - Obtain the latest nightly CLI from http://github.com/dotnet/cli
+    - Add a NuGet.Config to your app with the following content:
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+        <packageSources>
+            <clear />
+            <add key="AspNetCI" value="https://www.myget.org/F/aspnetcirelease/api/v3/index.json" />
```
Comment (moozzyk) [inline README.md]: shouoldn't this be aspnetrelease and not aspnetcirelease? My understanding is that aspnetcirelease contains untested packages...

Comment (pranavkm) [inline README.md]: or `aspnetrc2` which has specifially the rc2 escrow packages.

Comment (muratg) [inline README.md]: I recommend aspnetrelease... aspnetrc2 can potentially go away.

File: README.md [hunk 2]
```diff
@@ -1,97 +1,41 @@
...
 [![Join the chat at https://gitter.im/aspnet/Home](https://badges.gitter.im/aspnet/Home.svg)](https://gitter.im/aspnet/Home?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This guide is designed to get you started building applications with the latest development versions ASP.NET Core. This means nightly builds and potentially broken or unstable packages.
+**This is not the homepage for ASP.NET Core.** This is the home page of our source code repositories and is intended for those contributing to ASP.NET Core or using bleeding edge nightly builds.
 
-If you want a more stable, released, experience then you should go to https://www.asp.net/vnext.
+If you want a more stable, released, experience or getting started instructions then go to one of the following:
 
-## What you need
+- [.NET Landing Page](http://dot.net)
```
Comment (Eilon) [inline README.md]: What's a "landing page"? Use plain English...

File: README.md (hunk 1 above)
Comment (Eilon) [inline README.md]: Um, no, none of the above. Use aspnetvnext only. The aspnetrelease feed will be cleared out very soon once we release.

Comment (moozzyk) [inline README.md]: also this is talking about nightly builds - I don't think we will keep updating aspnetrc2 feed?

File: README.md (hunk 2 above)
Comment (glennc) [inline README.md]: Which of Landing and Page isn't plain english? ;)

File: README.md (hunk 1 above)
Comment (glennc) [inline README.md]: Will change to aspnetvnext.

File: README.md (hunk 2 above)
Comment (MaximRouiller) [inline README.md]: > What you need
> .NET Landing Page

Where do I install this Landing Page that you speak of. ;-)

Comment (glennc) [inline README.md]: Surrounded by comedians :). Changed it to Homepage just now.

**Comment by Eilon** [comment]
:shipit:

File: README.md (hunk 2 above)
Comment (MaximRouiller) [inline README.md]: @glennc If I made you laugh, my pleasure. Otherwise, well... at least I had a giggle. :stuck_out_tongue:

**Pull request merged**